config/config.go

```diff
@@ -190,24 +191,35 @@ func (c *Config) Validate() error {
 	return nil
 }
 
-// GetSourceIDs returns the Subnet and Chain IDs of all subnets configured as a source
-func (cfg *Config) GetSourceIDs() ([]ids.ID, []ids.ID, error) {
+// GetSourceIDs returns the Subnet, Chain IDs and Allowed Destination Chain IDs of all subnets configured as a source
+func (cfg *Config) GetSourceIDs() ([]ids.ID, []ids.ID, map[ids.ID]map[ids.ID]bool, error) {
```

Comment:
I know this is slightly out of scope for this feature, but could we have `GetSourceIDs` be a standalone function that returns the values from the global configuration singleton?

Reply:
Whoops, the change that adds a global config is still in a branch. I think we should register the config as a global singleton, rather than passing it through function calls.